modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/roundstartslime.dm

```diff
@@ -236,6 +237,27 @@
 			if(new_style)
 				alterer.facial_hairstyle = new_style
 				alterer.update_hair(is_creating = TRUE)
+		if("Hair Color")
+			var/hair_area = tgui_input_list(alterer, "Select which color you would like to change", "Hair Color Alterations", list("Hairstyle", "Facial Hair", "Both"))
```

Comment:
I prefer `tgui_alert` for short, fixed lists like this. It *works* as a `tgui_input_list`, but imo it's a bit cleaner as an alert.

Reply:
I guess you're right, the list won't really be changing anyway.